packages/turbo-telemetry/src/events/turbo-ignore.ts

```diff
@@ -0,0 +1,11 @@
+import { TelemetryClient } from "../client";
+import type { Event } from "./types";
+
+export class TurboIgnoreTelemetry extends TelemetryClient {
```

Comment:
Will extend this in a future PR, but added the stub in this PR to ensure the init generics work as expected.